apps/vendor/models.py

```diff
@@ -0,0 +1,168 @@
+from django.db import models
+import os
+from uuid import uuid4
+from apps.customer.models import LedgerAccounts
+from apps.masters.models import FirmStatuses, GstCategories, PriceCategories, Territory, Transporters
+
+# Create your models here.
+    
+class VendorCategory(models.Model):
+    vendor_category_id = models.AutoField(primary_key=True)
+    code = models.CharField(max_length=50, null=True, default=None)
+    name = models.CharField(max_length=255, null=True, default=None)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    def __str__(self):
+        return f'{self.vendor_category_id}_{self.name} ({self.code})'
+
+    class Meta:
+        db_table = 'vendor_category'
+
+
+class VendorPaymentTerms(models.Model):
+    payment_term_id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=255, null=True, default=None)
+    code = models.CharField(max_length=50, null=True, default=None)
+    fixed_days = models.IntegerField(null=True, default=None)
+    no_of_fixed_days = models.IntegerField(null=True, default=None)
+    payment_cycle = models.CharField(max_length=255, null=True, default=None)
+    run_on = models.CharField(max_length=255, null=True, default=None)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
```

Comment:
updated_at should satisfy below statement, did you test?
updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP